Xpirit-Vsts-Release-Terraform/terraform.ps1

```diff
@@ -134,9 +134,9 @@ function Initialize-Terraform
 
     if (-not ([string]::IsNullOrEmpty($additionalArguments)))
     {
-        $arguments = $remoteStateArguments + " $($additionalArguments.Trim()) $($extraArguments.Trim())"
+        $arguments = $remoteStateArguments + "$($extraArguments.Trim()) $($additionalArguments.Trim())"
     } else {
-        $arguments = $remoteStateArguments + " $($extraArguments.Trim())"
```

Comment:
you removed the extra spaces for which you add an extra commit to remove the trim. The space before the line is a better solution than removing the trim

Reply:
`extraArguments` is a constant with well-known value, we don't need to trim it because we know it always starts with a space and there's no way it might contain extra spaces in the end.